website/docs/components/code-editor/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,40 @@
+## Usage
+
+### When to use
+
+- To allow users to create and edit code with the support of syntax highlighting, change history, and keyboard navigation.
+
+### When not to use
+
+- To display static blocks of code.
```

Comment:
[Question] Should we add a link to the `CodeBlock` here, since that's the component that should be used in this situation?